textattack/goal_functions/classification/input_reduction.py

```diff
@@ -0,0 +1,36 @@
+from .classification_goal_function import ClassificationGoalFunction
+
+
+class InputReduction(ClassificationGoalFunction):
+    """
+    An targeted attack on classification models which attempts to maximize the 
```

Comment:
1. typo: 'A targeted'


2. maybe you should mention the Feng et al paper in this docstring? If I had stumbled upon this class without knowing about that paper, I might be very confused what its use is. Just an idea.

3. nit about this docstring: might be better if this says "... attempts to maximize the score of the target label. Complete when the target label is the predicted label. " or something like that-- I had to read this a couple times to understand

Reply:
I just forgot to change this from the TargetedClassification class LOL